### PR TITLE
fixing the begining;

### PR DESCRIPTION
**Pull request by xianwinwin**
can you please check and see if the beginning make sense?

### CODE REVIEW

File: my story.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-JUSTTTTTTTTTTTT
```
Comment (adhg) [inline my story.txt:1]: this is doable. can you make it as a song?

Comment (adhg) [reply my story.txt:1]: or a theme from a song?

File: my story.txt [hunk 2]
```diff
@@ -30,3 +30,4 @@ its adhg
 no, its xww
```
Comment (adhg) [inline my story.txt:30]: this can be removed!

Comment (xianwinwin) [reply my story.txt:30]: sure. just the 2 of us...we can make it if we try.

**Pull request merged**